kitchen.yml

```diff
@@ -21,6 +21,7 @@ platforms:
   - name: debian-10
   - name: fedora-29
```

Comment:
You can just change this to fedora-latest and we'll never need to update it again